### PR TITLE
Allow standalone activity creation

### PR DESCRIPTION
**Pull request by Mathieu414**
- Modify activity creation page
- Prepare /evenement page to have an "Ajouter une activité" button
- Update routes

### CODE REVIEW

File: app/pages/events/event_list/event_list.php [hunk 1]
```diff
@@ -44,8 +44,19 @@
...
+        <li>
+            <details class="dropdown">
+                <summary>Actions</summary>
+                <ul>
+                    <li><a href="/evenements/nouveau" class="secondary">
+                            <i class="fas fa-plus"></i> Ajouter un événement
+                        </a></li>
+                    <!-- <li><a href="/ajouter-activite" class="secondary">
+                            <i class="fas fa-plus"></i> Ajouter une activité
+                        </a></li> -->
```
Comment (Atmos4) [inline app/pages/events/event_list/event_list.php:56]: Can you rename routes to be more consistent: `ajouter-evenement` and `ajouter-activite`

File: app/pages/events/activity_edit.php [hunk 2]
```diff
@@ -3,14 +3,20 @@
...
+if ($event_id && !$event) {
     return "this event does not exist";
 }
 if ($activity_id) {
     $activity = em()->find(Activity::class, $activity_id);
+    if (!$activity) {
+        redirect($event_id ? "/evenements/$event_id/ajouter-activite" : "/ajouter-activite");
+    }
+    if ($activity->event->id != $event_id) {
+        redirect("/evenements/$activity->event->id/activite/$activity_id/modifier");
```
Comment (Atmos4) [inline app/pages/events/activity_edit.php:18]: This link might be broken
```suggestion
        redirect("/evenements/{$activity->event->id}/activite/$activity_id/modifier");
```

**Review by Atmos4 — COMMENTED** [review]
Very nice work 🙌

File: app/pages/events/event_list/event_list.php (hunk 1 above)
Comment (Mathieu414) [reply app/pages/events/event_list/event_list.php:56]: I renamed the `ajouter-activite` into `activite/nouveau`, seemed more consistent with the logic of the routes

File: app/pages/events/activity_edit.php (hunk 2 above)
Comment (Mathieu414) [reply app/pages/events/activity_edit.php:18]: Done

**Review by Atmos4 — APPROVED** [review]

**Pull request merged**